app/src/main/kotlin/cz/covid19cz/erouska/ui/help/data/FaqCategory.kt

```diff
@@ -10,3 +14,11 @@ data class FaqCategory(
     val icon: String,
     val questions: List<Question>
 ) : Category, Parcelable
+
+fun String?.toFaqCategories(): List<FaqCategory> {
+    val categoryType: Type = object : TypeToken<ArrayList<FaqCategory>>() {}.type
```

Comment:
we should try-catch this Gson block too

Reply:
what exactly do you mean by try-catching here?